src/ray/rpc/worker/core_worker_client_pool.cc

```diff
@@ -53,5 +53,21 @@ void CoreWorkerClientPool::Disconnect(ray::WorkerID id) {
   client_map_.erase(it);
 }
 
+void CoreWorkerClientPool::DisconnectAll() {
+  // disconnect
+  for (auto it : client_map_) {
+    auto conn = it->second;
+    rpc::ExitRequest request;
+    request.set_force_exit(true);
+    request.set_worker_id(it.first.Binary());
+    conn->Exit(request,
```

Comment:
This is not right: this will kill all the other workers.

Reply:
why should all workers be killed?

Reply:
I think this worker is over. It is a safe measure to disconnect all the workers who have established links.

Reply:
Yea currently worker needs to exit but we cannot kill all the other workers.